src/database/schema.js

```diff
@@ -188,7 +190,7 @@ export const schema = {
     User,
     Currency,
   ],
-  schemaVersion: 15,
+  schemaVersion: 17,
```

Comment:
what happened to poor old 16?

Reply:
Oops. Fixed (somewhere in the other commits!).